src/Core/src/Platform/iOS/TabbedViewExtensions.cs

```diff
@@ -9,6 +9,22 @@ namespace Microsoft.Maui.Platform
 {
 	internal static class TabbedViewExtensions
 	{
+		internal static void DisableiOS18ToolbarTabs(
+			this UITabBarController tabBarController)
+		{
+
+			bool is18 = OperatingSystem.IsMacCatalystVersionAtLeast(18);
```

Comment:
looks like this isn't used anywhere

Reply:
Whoops

Yea, this command is running incorrectly for me locally and I accidently left this in :-)